packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -0,0 +1,246 @@
+  "PaletteDangerLighter": "#fff0ee",
+  "PaletteDangerLight": "#fe8f7a",
+  "PaletteDangerMain": "#e72500",
+  "PaletteDangerDark": "#951800",
+  "PaletteWarningLighter": "#fcf6ac",
+  "PaletteWarningLight": "#eb9e05",
+  "PaletteWarningMain": "#a16c03",
+  "PaletteWarningDark": "#664402",
+  "PaletteSuccessLighter": "#defae7",
+  "PaletteSuccessLight": "#59c282",
+  "PaletteSuccessMain": "#16884a",
+  "PaletteSuccessDark": "#0e562f",
+  "PaletteNeutralMain": "#6e6e6e",
+  "PaletteNeutralDark": "#272727",
+  "ShadowScale0": "0px 1px 4px rgba(29, 29, 33, 0.08), 0px 4px 6px rgba(29, 29, 33, 0.01), 0px 5px 15px rgba(29, 29, 33, 0.05)",
+  "ShadowScale1": "0px 1px 4px rgba(29, 29, 33, 0.08), 0px 4px 10px rgba(29, 29, 33, 0.08), 0px 8px 30px rgba(29, 29, 33, 0.1)",
+  "Spacing0": "0",
+  "Spacing1": "0.28571429rem",
+  "Spacing2": "0.57142857rem",
+  "Spacing3": "0.85714286rem",
+  "Spacing4": "1.14285714rem",
+  "Spacing5": "1.71428571rem",
+  "Spacing6": "2.28571429rem",
+  "Spacing7": "2.85714286rem",
+  "Spacing8": "3.42857143rem",
+  "Spacing9": "4rem",
+  "TypographyColorBody": "#272727",
+  "TypographyColorHeading": "#272727",
+  "TypographyColorSubordinate": "#6e6e6e",
+  "TypographyColorDisabled": "#aeaeae",
+  "TypographyFamilyBody": "'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif",
+  "TypographyFamilyHeading": "'Aeonik', 'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif",
+  "TypographyFamilyButton": "'Aeonik', 'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif",
+  "TypographyFamilyMono": "'Roboto Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace",
+  "TypographySizeBase": "87.5%",
+  "TypographySizeSubordinate": "0.857rem",
+  "TypographySizeBody": "1rem",
+  "TypographySizeHeading6": "1.143rem",
+  "TypographySizeHeading5": "1.286rem",
+  "TypographySizeHeading4": "1.571rem",
+  "TypographySizeHeading3": "2.000rem",
+  "TypographySizeHeading2": "2.286rem",
+  "TypographySizeHeading1": "2.571rem",
+  "TypographyStyleNormal": "normal",
+  "TypographyWeightBody": "400",
+  "TypographyWeightBodyBold": "600",
+  "TypographyWeightHeading": "500",
+  "TypographyLineHeightBody": 1.5,
+  "TypographyLineHeightOverline": 1.3,
+  "TypographyLineHeightHeading6": 1.3,
+  "TypographyLineHeightHeading5": 1.3,
+  "TypographyLineHeightHeading4": 1.25,
+  "TypographyLineHeightHeading3": 1.25,
+  "TypographyLineHeightHeading2": 1.2,
+  "TypographyLineHeightHeading1": 1.2
+}
+```
+
+## Customization examples
+
+The following examples illustrate the impact of basic changes:
+
+### Color change
+
+* Button `#4C64E1` to `#D11DCA`
+* Text `E272727` to `#00297A`
+
+**Before color changes**
+
+<div class="half">
+
+![Your Alt Text](/img/siw-gen3/siw3-color-before.png)
```

Comment:
Do you typically leave the "Your Alt Text" placeholder?

Reply:
Good catch. Should be null, or have alt text. I've fixed it.